ci-operator/step-registry/openshift/upgrade/aws/single-node/openshift-upgrade-aws-single-node-workflow.yaml

```diff
@@ -0,0 +1,34 @@
+workflow:
+  as: openshift-upgrade-aws-single-node
+  steps:
+    allow_best_effort_post_steps: true
+    pre:
+    - ref: ipi-install-hosted-loki
+    - chain: ipi-conf-aws
+    - ref: single-node-conf
+    - chain: ipi-install-stableinitial
+    test:
+    - ref: openshift-e2e-test
+    post:
+    - chain: gather-network
+    - chain: gather-core-dump
+    - chain: ipi-aws-post
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: "release:initial"
+      OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE: "release:latest"
+    env:
+      TEST_TYPE: upgrade
+      TEST_SUITE: ""
+      TEST_UPGRADE_OPTIONS: ""
+      TEST_SKIPS: >-
+        \[sig-api-machinery\] Kubernetes APIs remain available for new connections
+        \[sig-api-machinery\] Kubernetes APIs remain available with reused connections
+        \[sig-api-machinery\] OpenShift APIs remain available for new connections
+        \[sig-api-machinery\] OpenShift APIs remain available with reused connections
+        \[sig-api-machinery\] OAuth APIs remain available for new connections
+        \[sig-api-machinery\] OAuth APIs remain available with reused connections
+        \[sig-network-edge\] Cluster frontend ingress remain available
+        \[sig-network-edge\] Application behind service load balancer with PDB is not disrupted
+        \[sig-imageregistry\] Image registry remain available
```

Comment:
I realize that `TEST_SKIPS` was my idea from #12233, but having lived with it in the proxy space for a while now, an important limitation is that it's a pain to clobber this per-job for tests which are fixed in one version of OpenShift but not in older versions.  So while I have no problem with this sort of thing in the short term, there should be a bug or bugs about adding skip logic to openshift/origin to avoid things we know will never work when there is only a single control-plane node (the Kube/OpenShift(/OAuth?) API stuff) or only one node period (the ingress/LB/registry stuff).  Ideally we create that bug (or bugs) before landing this PR, and link them from comments in the YAML.

Reply:
>there should be a bug or bugs about adding skip logic to openshift/origin

Good idea - filed https://bugzilla.redhat.com/show_bug.cgi?id=1947809